docs/docsite/rst/dev_guide/developing_collections_distributing.rst

```diff
@@ -4,11 +4,19 @@
 Distributing collections
 ************************
 
-You can distribute your collections by publishing them on a distribution server. Distribution servers include Ansible Galaxy, Red Hat Automation Hub, and privately hosted Automation Hub instances. You can publish any collection to Ansible Galaxy and/or to a privately hosted Automation Hub instance. If your collection is certified by Red Hat, you can publish it to the Red Hat Automation Hub.
+A collection is a distribution format for Ansible content. A typical collection contains modules and other plugins that address a set of related use cases. For example, a collection might automate administering a particular database. A collection can also contain roles and playbooks.
 
-Distributing collections involves three major steps:
-#. Configuring your distribution server(s)
-#. Building your collection artifact
+To distribute your collection and allow others to use it, you can publish your collection on one or more distribution servers. Distribution servers include:
+
+* Ansible Galaxy (accepts all collections)
```

Comment:
```suggestion
* Ansible Galaxy accepts all collections.
```
From the IBM style guide:

> Use parentheses to identify items such as abbreviations, symbols, and measurements, but avoid using
parentheses in general text.

Reply:
Thanks for the pointer @sagidlow. I'm not sure this the right change to make the doc follow the IBM style guide's guidance on parentheses here. The main point of this list is "here are the three distribution servers you can use", so turning each item into a sentence about which collections each one accepts feels odd. I might turn this into a table instead, with a column for `Distribution server` and another for `Content accepted`.

Reply:
Added a table in 2b84951fc61422551f159ccf5df686beff5fb23e.